pkgs/applications/misc/golden-cheetah/default.nix

```
@@ -73,6 +69,6 @@ in mkDerivation rec {
     description = "Performance software for cyclists, runners and triathletes";
     platforms = platforms.linux;
     maintainers = [ ];
-    license = licenses.gpl3;
+    license = licenses.gpl2Only;
```

```suggestion
    license = licenses.gpl2Plus;
```

See e.g. https://github.com/GoldenCheetah/GoldenCheetah/blob/cabe078453cbe8e136b8adf8d4187a5be878671b/src/Core/main.cpp#L4-L7.

---

good noticed, thx.

---

Feel free to make a separate PR for this.